api/src/wfl/executor.clj

```diff
@@ -247,6 +264,9 @@
                  (jdbc/update! tx details {:status status :updated now}
                                ["id = ?" id]))
                (sort-by :entity records))))]
+    (log/debug (format "%s Updating statuses for eligible workflows..."
+                       (log-prefix executor)))
+    ;; TODO: do we still need to update failed statuses in light of retry design?
```

Comment:
Flagging this for discussion.  A failed workflow itself wouldn't need to be reconsidered in the update loop, as we'll link it to its retry via the `retry` column.